cicd/deploy/config.ini

```diff
@@ -0,0 +1,174 @@
+WEB_SERVICE_ENABLED = true
+AUTH_COOKIE_DOMAIN = .ala.org.au
+USERDETAILS_JWT_ENABLED = true
+USERDETAILS_JWT_FALLBACK_TO_LEGACY = true
+AFFILIATION_SURVEY = true
+HEADER_AND_FOOTER_BASEURL = https://www.ala.org.au/commonui-bs3-2019
+HEADER_AND_FOOTER_VERSION = 2
+ENABLED_AUTHORISED_SYSTEMS = true
+ENABLED_BULKCREATE = false
+ENABLED_EXPORTUSERS = false
+OPENAPI_SERVER_URLS = https://api.ala.org.au/userdetails/
+TOKENAPP_URL =  https://tokens.ala.org.au/token
+DOCSPORTAL_URL = https://docs.ala.org.au/#introduction
+DEFAULT_LOGOUT_REDIRECT_URL = https://www.ala.org.au
+DYNAMIC_CLIENT_TOKENS_CALLBACKURLS = ["https://tokens.ala.org.au/login"]
+DYNAMODBTABLE_DYNAMODBPK = userId
+DYNAMODBTABLE_DYNAMODBSK = clientId
+APIKEY_AUTH_URL = https://auth.ala.org.au/apikey/ws/check/
+APIKEY_CHECK_URL = https://auth.ala.org.au/apikey/ws/check?apikey=
+APIKEY_USERDETAILS_URL = https://auth.ala.org.au/userdetails/userDetails/
+DIGIVOL_ENABLED = true
+SANDBOX_ENABLED = true
+BIOCOLLECT_ENABLED = true
+
+SIGHTINGS_URL = https://biocollect.ala.org.au/spotter
+SPATIAL_URL = https://spatial.ala.org.au?tool=log
+VOLUNTEER_URL = https://volunteer.ala.org.au/user/myStats
+LISTS_URL = https://lists.ala.org.au/speciesList/list
+BIOCACHE_SEARCH_BASE_URL = https://biocache.ala.org.au/occurrences/search
+BIOCACHE_SEARCH_URL = "https://biocache.ala.org.au/occurrences/search/?q=*%3A*&fq=assertion_user_id%3A"
+BIOCACHE_DOWNLOAD_URL = https://biocache.ala.org.au/download/myDownloads/
+MYDATA_URL = https://sandbox.ala.org.au/tempDataResource/myData
+ALERTS_URL = https://alerts.ala.org.au
+ALA_BASE_URL = https://www.ala.org.au
+BIE_BASE_URL =  https://bie.ala.org.au
+SANDBOX_BASE_URL =  https://sandbox.ala.org.au
+
+[development]
+# code pipeline
+PIPELINE_STACK_NAME = ala-${PRODUCT_NAME}-${PRODUCT_COMPONENT}-pipeline-${CLEAN_BRANCH}
+BASE_STACK_NAME = ala-${PRODUCT_NAME}-base-${CLEAN_BRANCH}
+SLACK_CHANNEL = zabbix-userdetails
+HOSTED_ZONE = dev.ala.org.au
+SUB_DOMAIN = ${PRODUCT_NAME}-${CLEAN_BRANCH}
+
+###
+# application vars
+OIDC_DISCOVERY_URI = https://cognito-idp.ap-southeast-2.amazonaws.com/ap-southeast-2_OOXU9GW39/.well-known/openid-configuration
+USERDETAILS_OIDC_LOGOUTURL = https://auth-secure.auth.ap-southeast-2.amazoncognito.com/logout
+AUTH_COOKIE_NAME = ALA-Auth-Test
+COGNITO_POOLID = ap-southeast-2_OOXU9GW39
+HEADER_AND_FOOTER_BASEURL = https://www-test-2023.ala.org.au/commonui-bs3-2019
+HEADER_AND_FOOTER_VERSION = 2
+OPENAPI_SERVER_URLS = https://api.test.ala.org.au/userdetails/cognito/
+OPENAPI_OAUTH_URL = https://auth-secure.auth.ap-southeast-2.amazoncognito.com/oauth2
+GATEWAY_USAGEPLANID = yuz7fr
+DYNAMIC_CLIENT_TOKENS_CALLBACKURLS = ["https://tokens.test.ala.org.au/login"]
+TOKENAPP_URL =  https://tokens.test.ala.org.au/token
+DOCSPORTAL_URL = https://docs.test.ala.org.au/#introduction
+USERS_DELEGATED_GROUP_NAMES = "ap-southeast-2_OOXU9GW39_AAF,ap-southeast-2_OOXU9GW39_Facebook,ap-southeast-2_OOXU9GW39_Google,ap-southeast-2_OOXU9GW39_SignInWithApple"
```

Comment:
UserPoolIds can be pulled from the cognito stack here too

Reply:
Fixed